ivadomed/utils.py

```diff
@@ -432,6 +432,7 @@ def segment_volume(folder_model, fname_image, fname_prior=None, gpu_number=0):
             e.g. spinal cord centerline, used to crop the image prior to segment it if provided.
             The segmentation is not performed on the slices that are empty in this image.
         gpu_number (int): Number representing gpu number if available.
+        options (dict):
```

Comment:
What do you think about adding `fname_prior` to `options`? It kind of makes sense to me as this file is quite option..al. But happy either ways.

Reply:
I agree! I integrated your suggestion here: 4fe58a0 and edf5d9b